src/apps/frontend/constants/index.ts

```diff
@@ -1,6 +1,7 @@
 const constant = {
   EMAIL_VALIDATION_ERROR: 'Please enter a valid email',
   PASSWORD_MIN_LENGTH: 8,
+  PASSWORD_MATCH_VALIDATION_ERROR: 'Passwords do not match',
```

Comment:
I understand ideally these should be provided by PM. Since there is a lack of it, I would recommend to Google - `How to write good error messages` (and or similar Google search) and see if we can self improve this.